Zoo/src/main/java/Zoo/Animal.java

```diff
@@ -0,0 +1,26 @@
+package Zoo;
+
+import java.util.ArrayList;
+
+public abstract class Animal {
+    protected String animalName;
+    protected String animalInfo;
+    protected static ArrayList<String> listOfAnimals = new ArrayList<>();
```

Comment:
I think this shouldn't be here. As Animal is inherited by all the types of animals (Birds, Fishes, Mammals) every one of them will also contain this list of animals (even if it's empty it will still be there) it doesn't really belong in the Animal class, it either deserves the separate class (Zoo or something) or can simply be created in the Main class

Reply:
Should I move this variable also? 
protected String animalName;
protected String animalInfo;
I moved ArrayList to other class, but can't understand why I need to move this 2 variable.